extra/pd~/pd~.c

```diff
@@ -87,6 +86,13 @@ void critical_exit(int z);
 #include "s_stuff.h"
 static t_class *pd_tilde_class;
 #define PDERROR pd_error(x,
+#else
+#ifdef _MSC_VER
+/* Sorry, MSP! */
+#define pd_snprintf _snprintf
```

Comment:
why does this require a macro and it cannot use the definition in s_stuff.h and the implementation in s_print.c?

Reply:
because this is the Max part :) the macro is the exact thing that we used to do throughout the Pd codebase.

Reply:
Uh makes sense, `pd~` always gets me. At first I thought you were leaving a comment there for MIller. 🤦  But clearly I didn't look at more context.

Reply:
> At first I thought you were leaving a comment there for MIller.

:-D